package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Remote new tab (third version)",
   "main": "src/main.js",
   "scripts": {
-    "start": "npm run clean && npm run watch:js & npm run watch:css",
-    "build": "npm run clean && npm run build:js && npm run build:css && npm run build:offline",
+    "setup": "npm run clean && npm run build:static",
+    "start": "npm run setup && npm run watch:js & npm run watch:css",
+    "build": "npm run setup && npm run build:js && npm run build:css && npm run build:offline",
     "clean": "rm -rf ./www",
+    "build:static": "cp -r src/static/* www",
```

Comment:
this didn't work for me... However, this does: `cp -r src/static www`

See error:

```
> cp -r src/static/* www

usage: cp [-R [-H | -L | -P]] [-f | -i | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-f | -i | -n] [-apvX] source_file ... target_directory

npm ERR! Darwin 15.2.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build:static"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! newtab@0.0.0 build:static: `cp -r src/static/* www`
npm ERR! Exit status 64
npm ERR!
npm ERR! Failed at the newtab@0.0.0 build:static script 'cp -r src/static/* www'.
npm ERR! This is most likely a problem with the newtab package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cp -r src/static/* www
npm ERR! You can get their info via:
npm ERR!     npm owner ls newtab
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/rlr/dev/remote-newtab/npm-debug.log
```